### ISSUE TITLE
Why Inception V3 retrained using tensorflow and keras on same dataset showing  different accuracy?

### ISSUE THREAD

**Issue by Rahul-Venugopal** [keras-team/keras#12712]
Hi ,

I am trying to retrain Inception V3 pretrained on ImageNet dataset. 

I have retrained **Inception V3 using Keras**(tensorflow backend) with following code : 

```
# SETUP MODEL
    CLASSES = 3
    base_model = InceptionV3(weights='imagenet', include_top=False)
    x=base_model.output
    x=GlobalAveragePooling2D()(x)
    preds=Dense(CLASSES,activation='softmax')(x) #final layer with softmax activation

    model=Model(inputs=base_model.input,outputs=preds)
    
    # transfer learning
    for layer in base_model.layers:
        layer.trainable = False
    
    model.compile(loss="categorical_crossentropy", optimizer='adam',metrics=["accuracy"])
    
    # train the network
    print("[INFO] training network...")
    H = model.fit_generator(
    	aug.flow(trainX, trainY, batch_size=BS),
    	validation_data=(testX, testY),
    	steps_per_epoch=len(trainX) // BS,
    	epochs=EPOCHS, verbose=1, callbacks=[csv_logger])
```
After training for 100 epochs , I get an accuracy between **85% to 90%** as shown in plot :

![training result using keras](https://user-images.githubusercontent.com/30955455/56507394-743d2080-6521-11e9-93ea-ebb7028cf115.png)

********    Tensorflow  **********

I have also retrained Inception V3 pretrained on ImageNet using Tensorflow following the link : 
[How to Retrain an Image Classifier for New Categories](https://www.tensorflow.org/hub/tutorials/image_retraining)
After 4000 Iterations , I am getting an accuracy between **93% to 96%** as shown in plot below : 
![Training result using Tensorflow ](https://user-images.githubusercontent.com/30955455/56507551-f594b300-6521-11e9-92b5-5f75a1f73993.png)

**Could anyone please help me to find the explanation for this reason or please point out the the mistake (if any) , for my interpretation.** 

Thanks 
Rahul

**Comment by jvishnuvardhan** [comment]
Please fill the issue template [here](https://github.com/keras-team/keras/issues/new/choose). Could you update them if they are relevant in your case, or leave them as N/A? Along with the template, please provide as many details as possible to find the root cause of the issue. Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
